app/models/taxon_name.rb

```diff
@@ -16,6 +16,8 @@ class TaxonName < ActiveRecord::Base
   validates_format_of :lexicon, with: /\A[^\/,]+\z/, message: :should_not_contain_commas_or_slashes, allow_blank: true
   validate :species_common_name_cannot_match_taxon_name
   validate :valid_scientific_name_must_match_taxon_name
+  validate :english_lexicon_if_exists
```

Comment:
This should have `on: :create`. Without that that none of the other properties of an existing "invalid" name could be updated, like `is_valid` or `position`.

Reply:
Good call! I'm sure there will be additional considerations that arise after inspecting the existing data.